src/Serilog.Sinks.Seq/SeqLoggerConfigurationExtensions.cs

```diff
@@ -60,7 +60,8 @@ public static class SeqLoggerConfigurationExtensions
             long? bufferFileSizeLimitBytes = null,
             long? eventBodyLimitBytes = 256 * 1024,
             LoggingLevelSwitch controlLevelSwitch = null,
-            HttpMessageHandler messageHandler = null)
+            HttpMessageHandler messageHandler = null,
+            long? errorFilesSizeLimitBytes = null)
```

Comment:
Just realised that the arg name here's no longer consistent with what we're calling them elsewhere in the code. Would `retainedInvalidPayloadsLimitBytes` work? (Trying to match the two param names further up in the arg list).